website/website/index.html

```diff
@@ -108,6 +108,7 @@ <h3>2. Choose data quality report.</h3>
 						<option value="RoadReport">Road Report</option>
 						<option value="MultilevelMappingSaturation">Multilevel Mapping Saturation</option>
 						<option value="MultilevelCurrentness">Multilevel Currentness</option>
+						<option value="FoodRelatedReport">Food Related Report</option>
```

Comment:
Do we want to have this report on the website?

Reply:
no, I removed it